Packages/com.pg649.creaturegenerator/Runtime/SettingsObjects/JointLimitOverrides.cs

```diff
@@ -0,0 +1,30 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+[CreateAssetMenu(fileName = "JointLimitOverrides", menuName = "PG649-CreatureGenerator/Joint Limit Overrides")]
+public class JointLimitOverrides : ScriptableObject
+{
+    [System.Serializable]
+    public class JointTypeLimit
+    {
+        public BoneCategory parentBone;
+        public BoneCategory childBone;
+        public int XAxisMin;
+        public int XAxisMax;
+        public int YAxisSymmetric;
+        public int ZAxisSymmetric;
+    }
+
+    public JointTypeLimit[] limitOverrides;
+
+    public Dictionary<(BoneCategory, BoneCategory), JointLimits> ToDict()
```

Comment:
I would construct a LimitTable here. That class only exists as an alias for that long Dictionary Type anyway and you can move the merging logic that is currently duplicated between the two generators into the LimitTable class as well.